scripts/gulp/tasks/bundle.ts

```diff
@@ -22,6 +22,10 @@ task('bundle:package:clean', () => del([`${paths.packageDist(packageName)}`], {
 
 const componentsSrc = [paths.packageSrc(packageName, '**/*.{ts,tsx}')];
 
+task('copy:readme', function () {
```

Comment:
This should go to `packages/fluentui/react-northstar/gulpfile.ts`, you can define an override there for `bundle:package:no-umd`.

Currently it will be executed for each Northstar package:

![image](https://user-images.githubusercontent.com/14183168/129048552-ca8a830b-682b-475f-b84c-b8ecef62565a.png)

Reply:
Done